Services/Navigation/classes/class.ilNavigationHistory.php

```diff
@@ -146,7 +146,7 @@ public function getItems() : array
                             $tree->isInTree($rec["ref_id"]) &&
                             (
                                 !$objDefinition->isPluginTypeName($rec["type"]) ||
-                                $component_repository->getPluginById($it["type"])->isActive()
+                                $component_repository->getPluginById($it["type"])->isActive()// TODO PHP8-REVIEW $it is based on the laste iteration above. I suggest to use an ecplit new variable for this case.
```

Comment:
`$it ` is based on the laste iteration above. I suggest to use an ecplit new variable for this case.